### DISCUSSION TITLE
change brush width on zoom

### DISCUSSION THREAD

**Discussion by ChangeS99** [fabricjs/fabric.js#7576] [Q&A]
So I created a cursor which is a fabric.Circle on a cursor canvas whose diameter is the freeDrawingBrush.width. I have another canvas where all the free drawing is done. So When i zoom in on the drawing canvas, I want a way to make freeDrawingBrush width to have the same size as the cursor( the cursor will remain the same size when zoomed in).

**Comment by asturur** [comment]
hi @ChangeS99 the zoom interactions aren't built in in fabricJS.
In the same part of your code where you execute the zoom, you should change the width of the brush.

**Reply by ChangeS99** [reply]
I am trying to do that. But the thing is, i want to find a way such that width of the brush won't scale up as i zoom in, it will remain the same size as the cursor. 
For eg. this fiddle
link: https://jsfiddle.net/w8gk5hcs/1/

**Reply by asturur** [reply]
In the same part of your code where you wrote the zoom code, write more code to divide the brush width by zoom value.

**Reply by ChangeS99** [reply]
it becomes too small if do that division.

**Reply by asturur** [reply]
probably because you are constantly dividing the current width by zoom.
You have to divide the slider value from html by zoom.
Remember to parseInt the value before dividing.

**Reply by ChangeS99** [reply]
Sorry for the late response. It worked. Thank you very much.
